controllers/datadogagent/feature/remoteconfig/feature.go

```diff
@@ -93,6 +117,13 @@ func (f *rcFeature) ManageClusterAgent(managers feature.PodTemplateManagers) err
 	}
 	managers.EnvVar().AddEnvVar(enabledEnvVar)
 
+	if f.enabled {
+		// Volume to create the Remote Config Database
+		// Mandatory as the cluster agent root FS is by default read only
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏‏

```suggestion
		// Mandatory as the cluster agent root FS is read only by default 
```